modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -5,6 +5,7 @@
 from __future__ import absolute_import
 
 import re  # noqa: F401
+from decimal import Decimal
```

Comment:
Please remove this per the above comment

Reply:
Done

Reply:
Can you resolve this conversation?